templates/screen-capture.php

```diff
@@ -14,6 +14,26 @@
 page.settings.resourceTimeout = <?php echo $timeout ?>;
 <?php endif ?>
 
+<?php if (isset($blockAnalytics) && $blockAnalytics == true) : ?>
+page.onResourceRequested = function(requestData, request) {
+	if ((/google-analytics\.com/gi).test(requestData['url'])){
+		request.abort();
+	}
+	//
+	if ((/track.hubspot.com/gi).test(requestData['url'])){
+		request.abort();
+	}
+	// Lucky Orange Tracking
+	if ((/luckyorange\.com/gi).test(requestData['url'])){
+		request.abort();
+	}
+	// Lucky Orange Tracking
```

Comment:
Thanks for the review, @MASNathan! I added comments for HubSpot and Google Analytics tracking. Was that what you were referring to?